src/pages/NewBlogPage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
```

Comment:
If this page is designed to create new blogs, its not what I wanted. I wanted you to use modals for creating and editing posts. If you don't know what modal is go look it up here: [React Modals](https://www.educative.io/answers/what-is-a-react-modal)

Reply:
I will make with modal but i made just this route to store that modal in it.